.github/prompts/az-cost-optimize.prompt.md

```
@@ -0,0 +1,305 @@
+   | summarize RequestCount=count() by AccountName, bin(TimeGenerated, 1d)
+   ```
+
+3. **Calculate Baseline Metrics**:
+   - CPU/Memory utilization averages
+   - Database throughput patterns
+   - Storage access frequency
+   - Function execution rates
+
+4. **VALIDATE CURRENT COSTS**: 
+   - Using the SKU/tier configurations discovered in Step 2
+   - Look up current Azure pricing at https://azure.microsoft.com/pricing/ or use `az billing` commands
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)